tree/30_generic_methods/command_execution.cf

```diff
@@ -20,7 +20,7 @@
 # @description Execute a command
 # @action
 #
-# @parameter command_name  Command name
+# @parameter command_name  Command to run
```

Comment:
paremeter will still be named "Command name"  if you keep command_name

Reply:
in fact all parameter names are 

replace "_" by " " and uppercase of first character